java/README.md

```diff
@@ -1,5 +1,30 @@
 # enry-java
 
+## Usage
+
+`enry-java` package is available thorugh [maven central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22enry-java%22),
+so it be used easily added as a dependency in various package management systems.
+Examples of how to handle it for most commons systems are included below,
+for other systems just look at maven central's dependency information.
+
+### Apache Maven
+
+```xml
+<dependency>
+    <groupId>tech.sourced</groupId>
+    <artifactId>enry-java</artifactId>
+    <version>1.0.0</version>
```

Comment:
maybe you should put ${enry_version} or something like that here, to avoid the need of change the readme on every new release.